src/Websites/Websites/Cmdlets/WebApps/PublishAzureWebApp.cs

```diff
@@ -45,6 +46,10 @@ public class PublishAzureWebAppCmdlet : WebAppOptionalSlotBaseCmdlet
         [Parameter(Mandatory = false, HelpMessage = "Run cmdlet in the background")]
         public SwitchParameter AsJob { get; set; }
 
+        [Parameter(Mandatory = false, HelpMessage = "Sets the timespan in Milliseconds to wait before the request times out.")]
```

Comment:
Update the Help message to "Configurable timeout in millseconds to wait for deployment operation to complete"

Reply:
Also add what the value of what the default time out value is.

Reply:
Changed the Help Message.